src/main/java/com/conveyal/gtfs/stats/StopStats.java

```diff
@@ -1,7 +1,239 @@
 package com.conveyal.gtfs.stats;
 
+import com.conveyal.gtfs.GTFSFeed;
+import com.conveyal.gtfs.model.Route;
+import com.conveyal.gtfs.model.Service;
+import com.conveyal.gtfs.model.StopTime;
+import com.conveyal.gtfs.model.Trip;
+import com.conveyal.gtfs.stats.model.TransferPerformanceSummary;
+import gnu.trove.list.TIntList;
+import gnu.trove.list.array.TIntArrayList;
+import org.mapdb.Fun;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
 /**
  * Created by landon on 9/2/16.
  */
 public class StopStats {
+
+    private GTFSFeed feed = null;
+    private FeedStats stats = null;
+    private RouteStats routeStats = null;
+
+    public StopStats (GTFSFeed f, FeedStats fs) {
+        feed = f;
+        stats = fs;
+        routeStats = stats.route;
+    }
+
+    public Set<Route> getRoutes (String stop_id) {
+        return feed.patterns.values().stream()
```

Comment:
No, definitely not a statistic.  Should we move this helper to GTFSFeed or do you think it's not worth keeping?